### DISCUSSION TITLE
是否考虑搭建静态web官方网站提高用户体验

### DISCUSSION THREAD

**Discussion by GaryHjy** [empjs/emp#92] [Ideas]
## 问题描述

是否考虑搭建静态web官方网站提高用户体验，包括cli使用方法，问题描述等。可以用dumi之类的搭建。

**Comment by ckken** [comment]
@dragonkuang @BoatGina @flyyuan

**Comment by flyyuan** [comment]
官方网站: https://efoxteam.github.io/emp-docs
文档待完善
